### DISCUSSION TITLE
Grouping roles in a "market" model

### DISCUSSION THREAD

**Discussion by JorgeAnzola** [spatie/laravel-permission#1717] [Q&A]
I thought I opened the issue previously, but it looks like I didn't 🤦🏼‍♀️

In my application I have the concept of markets, for example "latam" or "asia", each market has its own model. The challenge is that there's a super admin (This could be a normal role), with access to everything, and then an admin per market. Where an admin from Asia cannot view anything from Latam. I also have other roles, like "assessor" or "customer", all of them related to a market.

 I'm thinking about linking the role to the market model id, as I wouldn't like to do something like `admin_latam`, as suggested [here](https://github.com/spatie/laravel-permission/issues/1575). I'm asking first before (maybe) reinventing the wheel.
 
 I also found [this](https://github.com/spatie/laravel-permission/issues/1629) that could help, the app doesn't have multiple tenants, but maybe could be related?

Thanks!

**Comment by fredsal** [comment]
You can try https://github.com/spatie/laravel-permission/discussions/1782
